src/effects/VST3/VST3Effect.h

```diff
@@ -0,0 +1,153 @@
+/**********************************************************************
+
+  Audacity: A Digital Audio Editor
+
+  @file VST3Effect.h
+
+  @author Vitaly Sverchinsky
+
+  @brief Part of Audacity VST3 module
+
+**********************************************************************/
+
+
+#pragma once
+
+#include <pluginterfaces/gui/iplugview.h>
+#include <pluginterfaces/vst/ivstaudioprocessor.h>
+#include <public.sdk/source/vst/hosting/module.h>
+
+#include "EffectInterface.h"
+
+#include "SampleCount.h"
+#include "effects/Effect.h"
```

Comment:
Unnecessary #include is still there.

Remember `VST3Effect` does not inherit `Effect`, strange but true